test/parallel/test-cluster-worker-kill.js

```diff
@@ -29,7 +29,8 @@ if (cluster.isWorker) {
       worker_emitDisconnect: [1, "the worker did not emit 'disconnect'"],
       worker_emitExit: [1, "the worker did not emit 'exit'"],
       worker_state: ['disconnected', 'the worker state is incorrect'],
-      worker_suicideMode: [false, 'the worker.suicide flag is incorrect'],
+      worker_voluntaryExitMode: [false,
```

Comment:
Since the language makes more sense to begin with, this can just be `exitedVoluntarily`, I think.